src/routes/eventRoute.ts

```diff
@@ -0,0 +1,8 @@
+import { Router } from "express";
+// import EventsController from "../controllers/eventController";
+
+const router = Router();
+
+// router.get("/events", EventsController.getEvents);
```

Comment:
if this is file is for event specific routes, then the middleware defined in `server.ts` should be explicit to 
```
app.use("/events", eventRoutes)
```
and in this file

```
router.get("/", eventsController.getEvents);
```